lib/hunter/commands/show.rb

```diff
@@ -49,6 +49,7 @@ def run
           puts a.join("\t")
         else
           Table.from_nodes([node], buffer: buffer).emit
+          puts node.content
```

Comment:
I'd like this to be output in the scenario where `--plain` is used, too. Please could you add it to the plain output, converting it to JSON first?